station.localization.json

```diff
@@ -1378,7 +1378,7 @@
       "sv": "Rel",
       "fr": "Rel",
       "de": "Rel",
-	  "pl": ""
+      "pl": ""
```

Comment:
I noticed some changes in the begining of the each PL string. Tabs changes to spaces or something like this?